content/exchange.html

```diff
@@ -0,0 +1,141 @@
+    </p>
+    <p>
+        - there been more exposure to the exchange program;
+    </p>
+    <p>
+        - the exchange application process been easier;
+    </p>
+    <p>
+        - global events such as Covid and the Hong Kong unrest not been a thing.
+    </p>
+</div>
+
+<div class="full">
+    <span class="graph-title">Reasons for not going on exchange</span>
+    <div id="exchange-no-reasons"></div>
+</div>
+<div class="vb"></div>
+
+<h3>SE Abroad</h3>
+<p class="full">
+    <span class="big">
+        When asked whether they would suggest going on exchange to future SEs, 7
+        out of 8 said yes.
+    </span>
+    SE students who have gone on exchange usually went on a study term after their
+    2A one. A good amount went on exchange in their 3B term. All but one respondent
+    recommended that future SEs attempt going on exchange. All respondents went abroad
+    for only a single term. The sister universities SEs attended were all in Europe
+    or Asia.
+</p>
+<div class="vb"></div>
+<div class="full">
+    <span class="graph-title"> Exchange countries in which SEs studied </span>
+    <p class="full center">
+        <b> Interactive: </b> Click on a highlighted country to see the sister
+        schools and the number of people who attended.
+    </p>
+    <div id="exchange-countries-map"></div>
+    <div id="exchange-map-text" class="center"></div>
+</div>
+<!-- TODO: since the countries are so small, have a list on the side indicating which
+     schools that can also zoom the map -->
+<div class="vbx2"></div>
+
+<h5> The Exchange Experience </h3>
+<p class="full">
+    <span class="big">
+        "An invaluable opportunity to travel and to immerse yourself in a different
+        culture."
+    </span>
+    We asked our exchangees to provide us some insight into their exchange experience.
+    <span class="small">
+        Many of the sentences are directly quoted or rephrased from the original submissions.
+    </span>
+</p>
+<div class="vb"></div>
+
+<h6>What was the hardest thing about going on exchange?</h6>
+<p class="quote">
+    The paperwork and the course planning.
+    <br /><br />
+    The language barrier - the area is mostly French-speaking.
+    <br /><br />
+    Finding a coop for the next term, especially if the term start/end dates did
+    not match the typical UW start/end dates.
+    <br /><br />
+    Connecting with the locals and making friends in a meaningful capacity. Some
+    exchange schools have a more mature program than others where it is easier to
+    make friends. If I were to do exchange again I would opt for a school where a
+    lot of international students go and where the program is well established.
+    <br /><br />
+    Balancing schoolwork/wanting to graduate at the same time as the SE cohort with
+    taking advantage of being in a new country. Oh, and the timezone differences when
+    communicating with friends, family, group project members and NA interviewers
```

Comment:
NA?

Reply:
probably north american. i will change that! 👀